generators/entity/index.js

```diff
@@ -273,7 +273,14 @@ module.exports = EntityGenerator.extend({
         if (this.relationships.length > 0) {
             this.log(chalk.white('Relationships'));
             this.relationships.forEach(function (relationship) {
-                this.log(chalk.red(relationship.relationshipName) + ' ' + chalk.white('(' + _.upperFirst(relationship.otherEntityName) + ')') + ' ' + chalk.cyan(relationship.relationshipType));
+            	var validationDetails = '';
+                var relationshipValidate = _.isArray(relationship.relationshipValidateRules) && relationship.relationshipValidateRules.length >= 1;
```

Comment:
use simple checks, the below is sufficient, no need of `relationshipValidate ` and stuff
```
if (relationship.relationshipValidateRules && relationship.relationshipValidateRules.indexOf('required') !== -1) {

}
```